braze/src/main/java/com/rudderstack/android/integration/braze/BrazeIntegrationFactory.java

```diff
@@ -39,10 +39,36 @@
 import java.util.Set;
 
 public class BrazeIntegrationFactory extends RudderIntegration<Braze> {
+    // String constants
     private static final String BRAZE_KEY = "Braze";
     private static final String BRAZE_EXTERNAL_ID_KEY = "brazeExternalId";
+    private static final String AUTO_IN_APP_MESSAGE_REGISTER = "auto_in_app_message_register";
+    private static final String DEFAULT_CURRENCY_CODE = "USD";
+    private static final String REVENUE_KEY = "revenue";
+    private static final String CURRENCY_KEY = "currency";
+    private static final String DATA_CENTER_KEY = "dataCenter";
+    private static final String API_KEY = "appKey";
+    private static final String SUPPORT_DEDUP = "supportDedup";
+
+    // Array constants
+    private static final Set<String> MALE_KEYS = new HashSet<>(Arrays.asList("M",
+            "MALE"));
+    private static final Set<String> FEMALE_KEYS = new HashSet<>(Arrays.asList("F",
+            "FEMALE"));
+    private static final List<String> RESERVED_KEY_SET = Arrays.asList("birthday", "email", "firstName",
+            "lastName", "gender", "phone", "address");
```

Comment:
```suggestion
    private static final List<String> RESERVED_KEY_SET = Arrays.asList("birthday", "email", "firstname",
            "lastname", "gender", "phone", "address");
```

Refer [FIRSTNAME_KEY](https://github.com/rudderlabs/rudder-sdk-android/blob/8b2976023d9fca64bacf07285dbb3c62c261486d/core/src/main/java/com/rudderstack/android/sdk/core/RudderTraits.java#L58) and [LASTNAME_KEY](https://github.com/rudderlabs/rudder-sdk-android/blob/8b2976023d9fca64bacf07285dbb3c62c261486d/core/src/main/java/com/rudderstack/android/sdk/core/RudderTraits.java#L61).